scripts/generate-google3-docs/resource-reference/templates/vertexai_vertexaiindex.tmpl

```diff
@@ -0,0 +1,54 @@
+{{template "headercomment.tmpl" .}}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}{{ .Kind}}{% endblock %}
+{% block body %}
+{{template "alphadisclaimer.tmpl" .}}
+<table>
+<thead>
+<tr>
+<th><strong>Property</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Name</td>
+<td>Vertex AI</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Documentation</td>
+<td><a href="/vertex-ai/docs">/vertex-ai/docs</a></td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Name</td>
+<td>v1beta1/projects.locations.indexes</td>
```

Comment:
We usually do `v1beta1.projects.locations.indexes` so maybe update it for consistency?

Reply:
Done